src/blob_file_cache.cc

```diff
@@ -73,7 +73,9 @@ Status BlobFileCache::FindFile(uint64_t file_number, uint64_t file_size,
     if (db_options_.advise_random_on_open) {
       f->Hint(RandomAccessFile::RANDOM);
     }
-    file.reset(new RandomAccessFileReader(std::move(f), file_name));
+    file.reset(new RandomAccessFileReader(std::move(f), file_name, nullptr,
+                                          nullptr, 0, nullptr,
+                                          env_options_.rate_limiter, true));
```

Comment:
To be consistent with rocksdb, we should rate limit GC only, not user reads. Seems `BlobGCJob` currently call `NewBlobFileReader` directly, which does not go through `BlobFileCache`. Let's revert the change here.